tests/utests/data/test_diff.c

```diff
@@ -0,0 +1,671 @@
+/**
```

Comment:
There is no test for failure? The diff is really supposed to not fail for some kind of input?

Reply:
Added a test returning inval arg.